pyanaconda/payload/dnf/payload.py

```diff
@@ -710,7 +674,7 @@ def update_base_repo(self, fallback=True, checkmount=True):
 
         self._include_additional_repositories()
         self._disable_unwanted_repositories()
-        self._load_enabled_repositories()
+        self._validate_enabled_repositories()
```

Comment:
It seems repos are loaded when adding, and then here reloaded for validation? If so - how does the performance look? Are the metadata cached after first access, is cache enabled?

Reply:
I have extended the documentation of `load_repository`.